### ISSUE TITLE
UserGetRegistrationListRequest no attribute line/port

### ISSUE THREAD

**Issue by kikohnl** [nigelm/broadworks_ocip#13]
* Broadworks OCI-P Interface version:broadworks_ocip-nested_elements
* Python version: 9.3.0
* Operating System:Ubuntu 20.04.4 LTS

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

Trying to get user registration data.
UserGetRegistrationListRequest has attributes user_id (str) userID

### What I Did
def UserGetRegistrationListRequest(devicename):
    try:
        #result=api.command("UserGetRegistrationListRequest", user_id=devicename+"@voip.hawaiiantel.net")
        result=api.command("UserGetRegistrationListRequest", user_id=devicename)

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.

```
kotis@cdrarchive:~/broadworks_ocip-nested_elements$ ./sr.py 
8084429642 PolyOTT 0004F289D2B4 PolycomVVX-VVX_410-UA/5.8.3.2414_0004f289d2b4
Traceback (most recent call last):
  File "./sr.py", line 62, in <module>
    main()
  File "./sr.py", line 55, in main
    expiration=UserGetRegistrationListRequest(device.device_name)
  File "./sr.py", line 31, in UserGetRegistrationListRequest
    result=api.command("UserGetRegistrationListRequest", user_id=devicename+"@voip.hawaiiantel.net")
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/api.py", line 370, in command
    return self.receive_response()
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/api.py", line 259, in receive_response
    return self.decode_xml(content)
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/api.py", line 287, in decode_xml
    result = cls.build_from_etree_(api=self, element=element, extras=extras)
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/base.py", line 453, in build_from_etree_
    initialiser = cls.build_initialiser_from_etree_(
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/base.py", line 426, in build_initialiser_from_etree_
    initialiser[elem.name] = cls.build_from_node_(
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/base.py", line 379, in build_from_node_
    return cls.decode_table_(node)
  File "/home/kotis/broadworks_ocip-nested_elements/broadworks_ocip/base.py", line 337, in decode_table_
    type: NamedTuple = namedtuple(typename, columns)  # type: ignore
  File "/usr/lib/python3.8/collections/__init__.py", line 358, in namedtuple
    raise ValueError('Type names and field names must be valid '
ValueError: Type names and field names must be valid identifiers: 'line/port'

**Comment by kikohnl** [comment]
```
<?xml version="1.0" encoding="ISO-8859-1"?>
<BroadsoftDocument protocol="OCI" xmlns="C" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<userId xmlns="">Userid</userId>
    <command xsi:type="UserGetRegistrationListRequest" xmlns="">
        <userId>8088881613</userId>
    </command><command xsi:type="InternalSystemParameterGetRequest" xmlns=""/>
</BroadsoftDocument>

<?xml version="1.0" encoding="ISO-8859-1"?>
<BroadsoftDocument protocol="OCI" xmlns="C" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<userId xmlns="">Userid</userId>
    <command echo="" xsi:type="UserGetRegistrationListResponse" xmlns="">
        <registrationTable>
            <colHeading>Device Level</colHeading>
            <colHeading>Device Name</colHeading>
            <colHeading>Order</colHeading>
            <colHeading>URI</colHeading>
            <colHeading>Expiration</colHeading>
            <colHeading>Line/Port</colHeading>
            <colHeading>Endpoint Type</colHeading>
            <colHeading>Public Net Address</colHeading>
            <colHeading>Public Port</colHeading>
            <colHeading>Private Net Address</colHeading>
            <colHeading>Private Port</colHeading>
            <colHeading>User Agent</colHeading>
            <colHeading>Lockout Started</colHeading>
            <colHeading>Lockout Expires</colHeading>
            <colHeading>Lockout Count</colHeading>
            <colHeading>Access Info</colHeading>
            <colHeading>Path Header</colHeading>
            <colHeading>Registration Active</colHeading>
            <row>
                <col>Group</col>
                <col>8088881613</col>
                <col>1</col>
                <col>sip:8088881613@10.13.6.xx:5060;dtg=UC1_BVE_INT_5061_TG;reg-info=39400</col>
                <col>Fri Jul 08 10:36:19 HST 2022</col>
                <col>8088881613@voip.hawaiiantel.xxx</col>
                <col>Primary</col>
                <col/>
                <col/>
                <col/>
                <col/>
                <col>PolyEdge-Edge_E220-UA/8.0.0.13670_48256712b942</col>
                <col/>
                <col/>
                <col>0</col>
                <col/>
                <col/>
                <col/>
            </row>
        </registrationTable>
    </command>
</BroadsoftDocument>

**Comment by kikohnl** [comment]
what else can i provide to move this issue along?

**Comment by nigelm** [comment]
Fixed in a8fe66b791e90024e2d884c567985818ce2e96e3

Now changes anything that isn't an alphanumeric character into a _ - multiple non-alphanumeric characters become a single _

**Comment by kikohnl** [comment]
Thank you this fixed my issue.
